src/test/java/org/jboss/hal/testsuite/test/util/ConfigAreaUtils.java

```diff
@@ -0,0 +1,165 @@
+package org.jboss.hal.testsuite.test.util;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.hal.testsuite.fragment.ConfigAreaFragment;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.formeditor.Editor;
+import org.jboss.hal.testsuite.page.ConfigPage;
+import org.jboss.hal.testsuite.util.ResourceVerifier;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author mkrajcov <mkrajcov@redhat.com>
+ */
+public class ConfigAreaUtils {
```

Comment:
I would prefer name of class which doesn't contain Utils as this is not standard utility class. E.g. something like `ConfigAreaChecker` or something else